vhdbuilder/packer/install-dependencies.sh

```diff
@@ -377,7 +377,7 @@ done
 
 CALICO_POD2DAEMON_IMAGES="
 v3.8.0
-v3.8.9
```

Comment:
why do we remove 3.8.9 here but keep it in other places?

Reply:
Because I fatfingered it. Sending fix.